### PR TITLE
PUBDEV-8067: Add info about ability to export xlsx file from new addition .feature_interaction()

### PR DESCRIPTION
**Pull request by hannah-tillman**
For: https://h2oai.atlassian.net/browse/PUBDEV-8067

Also fixed the spacing for path param in model_base.py.

### CODE REVIEW

File: h2o-py/h2o/model/model_base.py [hunk 1]
```diff
@@ -421,9 +421,10 @@ def feature_interaction(self, max_interaction_depth=100, max_tree_depth=100, max
         :param max_interaction_depth: Upper bound for extracted feature interactions depth. Defaults to 100.
         :param max_tree_depth: Upper bound for tree depth. Defaults to 100.
         :param max_deepening: Upper bound for interaction start deepening (zero deepening => interactions 
-        starting at root only). Defaults to -1.
-        :param path: Path where to save the output in .xlsx format. Please note that Pandas and XlsxWriter need to be 
-        installed for using this option. Defaults to None.
+            starting at root only). Defaults to -1.
+        :param path: Path where to save the output in .xlsx format (e.g. ``/mypath/file.xlsx``).
```
Comment (neemah2o) [inline h2o-py/h2o/model/model_base.py:425]: Add that it is optional. Like ":param path: (Optional )Path"

File: h2o-r/h2o-package/R/models.R [hunk 2]
```diff
@@ -2264,6 +2264,8 @@ h2o.get_ntrees_actual <- function(object) {
 #' @param max_tree_depth Upper bound for tree depth. Defaults to 100.
 #' @param max_deepening Upper bound for interaction start deepening (zero deepening => interactions 
 #' starting at root only). Defaults to -1.
+#' @param path Path where to save the output in .xlsx format (e.g. "/mypath/file.xlsx"). Please note
```
Comment (neemah2o) [inline h2o-r/h2o-package/R/models.R:2267]: Add that it is optional, "@param path (Optional) Path "

**Review by neemah2o — CHANGES_REQUESTED** [review]
Add that it is optional. Otherwise, looks good

**Review by koniecsveta — APPROVED** [review]
LGTM

**Review by neemah2o — APPROVED** [review]

**Pull request merged**